test/unit/krane/template_sets_test.rb

```diff
@@ -32,6 +33,16 @@ def test_template_sets_with_invalid_suffix_is_invalid
     assert_equal(template_sets.validate, expected)
   end
 
+  def test_template_sets_with_erb_files_are_considered_invalid_when_render_erb_is_false
+    bad_filepath = File.join(fixture_path("for_unit_tests"), "partials_test.yaml.erb")
+    template_sets = template_sets_from_paths(bad_filepath, render_erb: false)
+    expected = [
+      "Rendering erb during deploy is not supported. " \
+        "Try piping the output of `krane render` to `krane-deploy` with the --stdin flag",
+    ]
+    assert_equal(template_sets.validate, expected)
```

Comment:
I'm ok with this for now, but would prefer if, in the near future, we change this to be more in line with the 
```
File #{filename} does not have valid suffix (supported suffixes: ...
```
error (along with the hint to use `krane render`, to maintain consistency and also to enumerate the 'bad' files to the user

Reply:
I'm here now, I'll just do it now!